pages/docs/reference/collection-write.md

```diff
@@ -28,14 +28,7 @@ fun main() {
 ```
 </div>
 
-[`addAll()`](/api/latest/jvm/stdlib/kotlin.collections/add-all.html) adds every element of the argument object to a list or a set.
```

Comment:
I believe these two removed paragraphs are rephrased below.

Reply:
@p7nov please help choosing which one to leave.

Reply:
Yep, this is correct.  Seems like I forgot to remove the old description when rewriting it.